sims/benchmark-simuPOP.py

```diff
@@ -237,31 +234,37 @@ def __call__(self, loc, alleles):
 rc = RecombCollector(ts=init_ts, node_ids=node_ids,
                      locus_position=locus_position,
                      benchmark=True, mode='binary')
-
+pre_ops = [sim.PyOperator(lambda pop: rc.increment_time() or True)]
+mating_ops = [id_tagger]
+post_ops = []
+if args.record_neutral:
+    pre_ops += [sim.SNPMutator(u=args.mut_rate, v=args.mut_rate)]
+    #                               loci=neutral_loci)]
+    mating_ops += [sim.Recombinator(rates=args.recomb_rate,
+                                    infoFields="ind_id")]
+elif not args.record_neutral:
+    mating_ops += [sim.Recombinator(rates=args.recomb_rate,
+                                    output=sim.WithMode(rc.collect_recombs, 'b'),
+                                    infoFields="ind_id")]
+    post_ops += [sim.PyOperator(
+        lambda pop: rc.simplify(pop.indInfo("ind_id")) or True,
+        step=args.simplify_interval)]
 pop.evolve(
     initOps=[
         sim.InitSex(),
     ]+init_geno,
-    preOps=[
-        sim.PyOperator(lambda pop: rc.increment_time() or True),
+    preOps=pre_ops + [
         sim.SNPMutator(u=args.sel_mut_rate, v=args.sel_mut_rate),
+        #                   loci=selected_loci),
         # so that selector returns, for f_i fitness values, \prod_i f_i
         sim.PyMlSelector(GammaDistributedFitness(shape=args.gamma_shape,
```

Comment:
How does `PyMlSelector()` know which mutations are selected and which are neutral?